Sources/PostmarkClient/MailClient.swift

```diff
@@ -0,0 +1,74 @@
+import HTTP
+import Mail
+import Vapor
+
+
+public final class MailClient {
+    
+    let serverToken: String
+    let client: ClientProtocol
+    public var result : Response?
```

Comment:
I've exposed this `result` here for testing purposes, but I don't intend to keep it here in the end.